submissions/yevheniiii/html_css_popup/index.html

```diff
@@ -0,0 +1,147 @@
+
+            <section class="nav-item"><a href="#">Images</a></section>
+
+            <input type="checkbox" id="popup-btn" class="pop-up" />
+            <label for="popup-btn" class="button"></label>
+
+            <section class="iconsContainer">
+                <ul class="list-items">
+                    <li class="inside-item">
+                        <a href="#">
+                            <img src="images/drive.png" alt="drive" />
+                            <section class="items-description">Drive</section>
+                        </a>
+                    </li>
+
+                    <li class="inside-item">
+                        <a href="#">
+                            <img src="images/calendar.png" alt="calendar" />
+                            <section class="items-description">Calendar</section>
+                        </a>
+                    </li>
+
+                    <li class="inside-item">
+                        <a href="#">
+                            <img src="images/drive.png" alt="drive" />
+                            <section class="items-description">Drive</section>
+                        </a>
+                    </li>
+
+                    <li class="inside-item">
+                        <a href="#">
+                            <img src="images/business.png" alt="business" />
+                            <section class="items-description">Business</section>
+                        </a>
+                    </li>
+
+                    <li class="inside-item">
+                        <a href="#">
+                            <img src="images/translator.jpg" alt="translator" />
+                            <section class="items-description">Translator</section>
+                        </a>
+                    </li>
+
+                    <li class="inside-item">
+                        <a href="#">
+                            <img src="images/music.png" alt="music" />
+                            <section class="items-description">GMusic</section>
+                        </a>
+                    </li>
+
+                    <li class="inside-item">
+                        <a href="#">
+                            <img src="images/plus.png" alt="gplus" />
+                            <section class="items-description">GPlus</section>
+                        </a>
+                    </li>
+
+                    <li class="inside-item">
+                        <a href="#">
+                            <img src="images/bell.png" alt="bell" />
+                            <section class="items-description">Bell</section>
+                        </a>
+                    </li>
+
+                    <li class="inside-item space"></li>
+                </ul>
+
+                <input type="checkbox" id="buttonMore" class="MoreApps" />
+                <label for="buttonMore" class="MorePopUp">More</label>
+
+                <hr class="line" />
```

Comment:
For line better use elements border. 
Avoid to clogging up the markup by unnecessary tags.

Reply:
I removed the tag from HTML and added it as element border. Thanks!